<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewContactDialog</class>
 <widget class="QDialog" name="NewContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>541</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>541</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>通信簿信息输入</string>
  </property>
  <widget class="QGroupBox" name="box_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>个人通信信息</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_name">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,8">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_sex">
         <property name="text">
          <string>性别：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_sex">
         <item>
          <property name="text">
           <string>男</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>女</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,8">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_fphone">
         <property name="text">
          <string>家庭电话：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_fphone">
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,8">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_ophone">
         <property name="text">
          <string>办公电话：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_ophone">
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,8">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_mphone">
         <property name="text">
          <string>手机：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_mphone">
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,8">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_short">
         <property name="text">
          <string>短号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_short">
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,8">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_qq">
         <property name="text">
          <string>QQ：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_qq">
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,8">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_wechat">
         <property name="text">
          <string>微信号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_wechat">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,8">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_email">
         <property name="text">
          <string>电子邮箱：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_email">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,8">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_unit">
         <property name="text">
          <string>单位：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_unit">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="box_remark">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>521</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>备注</string>
   </property>
   <widget class="QPlainTextEdit" name="edit_remark">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>NewContactDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
